src/native/NativeInstabug.ts

```diff
@@ -136,12 +136,29 @@ export interface InstabugNativeModule extends NativeModule {
   addFileAttachmentWithURLToReport(url: string, filename?: string): void;
   addFileAttachmentWithDataToReport(data: string, filename?: string): void;
   willRedirectToStore(): void;
+
+  // W3C Feature Flags
+  isW3ExternalTraceIDEnabled(): Promise<boolean>;
+
+  isW3ExternalGeneratedHeaderEnabled(): Promise<boolean>;
+
+  isW3CaughtHeaderEnabled(): Promise<boolean>;
+
+  // W3C Feature Flags Listener for Android
+  registerW3CFlagsChangeListener(
+    handler: (payload: {
+      isW3ExternalTraceIDEnabled: boolean;
+      isW3ExternalGeneratedHeaderEnabled: boolean;
+      isW3CaughtHeaderEnabled: boolean;
+    }) => void,
+  ): void;
 }
 
 export const NativeInstabug = NativeModules.Instabug;
 
 export enum NativeEvents {
   PRESENDING_HANDLER = 'IBGpreSendingHandler',
+  ON_W3C_FLAGS_CHANE = 'IBGCoreOnNewW3CFlagsUpdateReceivedCallback',
```

Comment:
Let's rename this one as well if you think we should rename the one in the native.

```suggestion
  ON_W3C_FLAGS_CHANE = 'IBGOnNewW3CFlagsUpdateReceivedCallback',
```